rollup.config.js

```diff
@@ -50,9 +51,12 @@ function createDeclarationConfig(input, output) {
 function createESMConfig(input, output) {
   return {
     input,
-    output: { file: output, format: 'esm' },
+    output: [
+      { file: `${output}.js`, format: 'esm' },
+      { file: `${output}.mjs`, format: 'esm' },
```

Comment:
wow, we don't even need rollup copy plugin.

Reply:
we can still use it for the type definition copying if we want to get rid of the scripts in `package.json` https://github.com/pmndrs/zustand/pull/546

Reply:
ah, that's right..